README.md

```diff
@@ -38,10 +39,6 @@ Through Github
 - Linter test in local:
   `npx stylelint "**/*.{css,scss}"`
 
-### Deployment
-
-- Not deployed yet
-
 ## Authors
 
 👤 **Author1**
```

Comment:
- [ ] OPTIONAL
Generally, the subheading text Author1 should be replaced with your full name in your ReadMe.